src/main/java/uk/gov/companieshouse/siccode/api/search/CombinedSicActivitiesStorageModel.java

```diff
@@ -104,48 +108,62 @@ public int hashCode() {
         result = prime * result
                 + ((activityDescriptionSearchField == null) ? 0 : activityDescriptionSearchField.hashCode());
         result = prime * result + ((id == null) ? 0 : id.hashCode());
-        result = prime * result + (isCompaniesHouseactivity ? 1231 : 1237);
+        result = prime * result + (companiesHouseactivity ? 1231 : 1237);
         result = prime * result + ((sicCode == null) ? 0 : sicCode.hashCode());
         result = prime * result + ((sicDescription == null) ? 0 : sicDescription.hashCode());
         return result;
     }
 
     @Override
     public boolean equals(Object obj) {
```

Comment:
This whole method (class?) needs formatting, as some of the indents in this if chain are wrong.